components/atomic/buttons/ButtonControl/ButtonControl.styles.ts

```diff
@@ -78,3 +79,8 @@ export const ButtonText = styled.span<Pick<BaseProps, "size" | "icon">>`
       padding-inline-end: ${pxToRem("10px")};
     `}
 `;
+
+export const Disclosure = styled(DialogDisclosure)`
+  --dropdown-list-item-padding: ${basePadding(2)}, ${basePadding(6)};
+  --button-background: none;
```

Comment:
@1aurend This line is what's removing the button background from the delete button, and looks like it can be removed 👍